src/theme/components/menu.ts

```diff
@@ -0,0 +1,113 @@
+import { SectionTypes } from "../../helpers/types";
+import { getSectionColors } from "../../utils/getSectionColors";
+interface MenuStyleProps {
+  highlightColor: SectionTypes;
+  showBorder: boolean;
+  showLabel: boolean;
+}
+
+const Menu = {
+  parts: ["menuItem", "menuButton", "menuGroup", "menuList"],
+  baseStyle: ({
+    highlightColor = "blogs",
+    showBorder,
+    showLabel,
+  }: MenuStyleProps) => {
+    const sectionColor = getSectionColors({
+      type: highlightColor,
+      colorValue: highlightColor === "blogs" ? "secondary" : "primary",
```

Comment:
@bigfishdesign13 There's no design token for section colors with 5% opacity, so currently I don't have the background of the selected item set. I tried a couple CSS workarounds that did not work mainly bc they're not passed as rgb values– is there a plan to add these colors to the DS

Reply:
That token, and a whole array of related tokens, is being completed in the work I am doing for `StatusBadge`.

https://github.com/NYPL/nypl-design-system/pull/1507

Reply:
Small issue, the `StatusBadge` PR is for the React18 branch and this PR is for the `development` branch.

Can we live without the proper design token value for now until the `Menu` component is promoted to the new version?

Reply:
Adding them to this PR sounds good to me. When merging upstream, we should just watch out for any conflicts.

Reply:
@bigfishdesign13 sounds good!